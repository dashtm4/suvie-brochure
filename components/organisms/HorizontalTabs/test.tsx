import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { HorizontalTabs } from '.'

describe('<HorizontalTabs />', () => {
  const selectedIndex = 0
  const children = 'Test children'
  const onSelect = jest.fn()

  it('should renders correct <HorizontalTabs />', () => {
    const wrapper = shallow(
      <HorizontalTabs selectedIndex={selectedIndex} onSelect={onSelect}>
        {children}
      </HorizontalTabs>
    )

    expect(wrapper.contains(children)).toBeTruthy()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
