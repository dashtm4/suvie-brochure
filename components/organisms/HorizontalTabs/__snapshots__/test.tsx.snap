// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HorizontalTabs /> should renders correct <HorizontalTabs /> 1`] = `
<ForwardRef(LoadableComponent)
  className=""
  onSelect={[Function]}
  selectedIndex={0}
  selectedTabClassName="text-opacity-100 border-b-4 border-primary-default"
  selectedTabPanelClassName="px-3 md:px-0"
>
  Test children
</ForwardRef(LoadableComponent)>
`;
